app/controllers/application_controller.rb

```diff
@@ -455,6 +458,13 @@ def maintenance_warning
     @minutes_to_maintenance = NextMaintenance.minutes_to(now)
   end
 
+  def get_onboarding_status
+    if @is_current_community_admin
+      #TODO: maybe not with global variable?
```

Comment:
This comment feels like a good idea to me. Instead of doing this with filter it's enough if get_onboarding_status is exposed as a method and called in application.haml when topbar is initialized.